google/cloud/ndb/_datastore_api.py

```diff
@@ -85,9 +86,9 @@ def rpc_call():
         context = context_module.get_toplevel_context()
 
         call = method.future(request, timeout=timeout)
-        rpc = _remote.RemoteCall(call, "{}({})".format(rpc_name, request))
```

Comment:
Passing `request` here also sneakily caused a bunch of protocol buffers to get rendered as strings. It wasn't a logging statement, but is the same class of performance problem so it is included here.